build/DependencyVersions.props

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <MicrosoftAspNetCoreAllPackageVersion>2.1.0-preview1-28042</MicrosoftAspNetCoreAllPackageVersion>
```

Comment:
This name doesn't look right: "MicrosoftAspNetCoreAllPackageVersion" the "All" looks out of place.
That means the binary is named "Microsoft.AspNet.Core.All.dll" - is that accurate?

Reply:
That's the package name. That's accurate.

Reply:
It means it uses the Microsoft.AspNetCore.All metapackage to determine the version of aspnetcore. I changed this from Microsoft.AspNetCore.App because that package isn't ready yet (pending more design and discussion: https://github.com/aspnet/Universe/issues/721 )

Reply:
tks